src/test/groovy/com/github/rodionovsasha/commentservice/controllers/TopicControllerTest.groovy

```diff
@@ -166,11 +167,44 @@ class TopicControllerTest extends Specification {
         }
     }
 
+    def "#getActiveTopic returns active topic by id"() {
+        when:
+        def response = getActiveTopic(TOPIC_ID)
+
+        then:
+        1 * service.getActiveTopic(TOPIC_ID) >> topic
+        with(response) {
+            status == HttpStatus.OK.value()
```

Comment:
Interesting if spock allows to create a custom assertion for 
```groovy
status == HttpStatus.OK.value()
contentType == APPLICATION_JSON_UTF8_VALUE
```

Or if we can make a utility method like
```groovy
def extractJson(response, status = HttpStatus.OK) {
  assert response.status == status.value()
  assert contentType == APPLICATION_JSON_UTF8_VALUE
  getJsonFromString(response.contentAsString)
}
```

PS: Not a PR blocker, just a question